src/client/common/installer/productInstaller.ts

```diff
@@ -66,9 +114,12 @@ export abstract class BaseInstaller {
     public async install(
         product: Product,
         resource?: InterpreterUri,
-        cancel?: CancellationToken
+        cancel?: CancellationToken,
+        reInstallAndUpdate?: boolean
     ): Promise<InstallerResponse> {
-        return this.serviceContainer.get<IPythonInstaller>(IPythonInstaller).install(product, resource, cancel);
+        return this.serviceContainer
+            .get<IPythonInstaller>(IPythonInstaller)
+            .install(product, resource, cancel, reInstallAndUpdate);
```

Comment:
I think this flag needs to be added to vscode-python too, because `IPythonInstaller` wraps the contributed installer from the Python extension?

Reply:
Ah yes you mentioned that in the issue description